[CF1] private IP and BISO limit (#22244)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/policies/browser-isolation/known-limitations.mdx

| Precedence | Selector | Operator | Value                                | Action  |
| ---------- | -------- | -------- | ------------------------------------ | ------- |
| 2          | Host     | in       | `your-salesforce-domain.example.com` | Isolate |

## Browser Isolation is not compatible with private IPs on non-`443` ports

> Browser Isolation is not compatible with [self-hosted private applications](/cloudflare-one/applications/non-http/self-hosted-private-app/) that use private IP addresses on ports other than `443`. Trying to access an application with this configuration will result in a Gateway block page.

Reviewer: with self-hosted applications defined by private ips on ports other than 443

> Browser Isolation is not compatible with [self-hosted private applications](/cloudflare-one/applications/non-http/self-hosted-private-app/) that use private IP addresses on ports other than `443`. Trying to access an application with this configuration will result in a Gateway block page.

Author: ```suggestion
Browser Isolation is not compatible with [self-hosted private applications](/cloudflare-one/applications/non-http/self-hosted-private-app/) that use private IP addresses on ports other than `443`. Trying to access self-hosted applicationd defined by private IPs on ports other than `443` will result in a Gateway block page.
```

Author: @GreenStage how does it look?

Reviewer: lgtm

> Browser Isolation is not compatible with [self-hosted private applications](/cloudflare-one/applications/non-http/self-hosted-private-app/) that use private IP addresses on ports other than `443`. Trying to access self-hosted applicationd defined by private IPs on ports other than `443` will result in a Gateway block page.

Author: ```suggestion
Browser Isolation is not compatible with [self-hosted private applications](/cloudflare-one/applications/non-http/self-hosted-private-app/) that use private IP addresses on ports other than `443`. Trying to access self-hosted applications defined by private IPs on ports other than `443` will result in a Gateway block page.
```
